kolibri/plugins/learn/assets/src/views/ResourceSyncingUiAlert.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+
+  <MissingResourceAlert
+    v-if="isSyncing || syncCouldNotComplete"
```

Comment:
Oh, sorry - I'm still confused by this `v-if` why are we conditionalizing displaying the MissingResourceAlert on anything?

Reply:
because otherwise, the missing resource alert shows up in the "interim" and we don't actually solve the problem that Radina reported. So, a sync starts, the lesson structure syncs but maybe not all of the resources have, the "missing resource alert" shows up that there are resources missing, but the resources might be in progress of it syncing

Reply:
maybe this condition could be better handled elsewhere, but I am not sure how else to get around this

Reply:
If you want this condition to cover this behaviour it would have to be `v-if="!isLearnerOnlyImport || isSyncing || syncCouldNotComplete"` - although, to be honest, I'm not sure why we would want to not alert that resources are missing when resources are missing?

The only conditional here should be about reporting about _why_ they are missing if we have any additional information.